tasks/main.yml

```diff
@@ -6,5 +6,8 @@
 - include: tools.yml
   when: galaxy_tools_install_tools
 
+- include: workflows.yml
+  when: galaxy_tools_workflows|length > 0
```

Comment:
I'd say we should have that variable. This is shorter but having the variable is more consistent with the rest of the code and also if someone has workflows listed, they will not have to comment them out.